charts/ceph-csi-cephfs/values.yaml

```diff
@@ -33,7 +33,10 @@ csiConfig: []
 # Set logging level for csi containers.
 # Supported values from 0 to 5. 0 for general useful logs,
 # 5 for trace level verbosity.
+# logLevel is the variable for driver/plugin containers's log level
 logLevel: 5
+# sidecarLogLevel is the variable for sidecar container's log level
```

Comment:
`sidecarLogLevel is the  for kubernetes sidecar container's log level`

Reply:
yeah, reworded..